src/ensembl/src/content/app/custom-download/components/checkbox-grid/CheckboxGrid.scss

```diff
@@ -0,0 +1,15 @@
+@import 'src/styles/common';
+
+.checkboxGridContainer {
+  width: 100%;
+  display: flex;
```

Comment:
is this flex used for anything (e.g. is it better than `display: inline-block; width: 100%`)?

_Also, I am not quite sure I understand why `display: block` doesn't work here (tried in my browser)._

Reply:
I have used `flex` so that the columns inside this container gets displayed horizontally.
Is there any particular reason to avoid using it?